metastore-grpc-proxy/BUILD

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
+
+py_test(
```

Comment:
You need to include test in the test suite: https://github.com/GoogleCloudDataproc/initialization-actions/blob/dc0185effc7800218da11fbce3c355ed05498815/BUILD#L3-L42

Reply:
Done, thanks